.travis/test_bindings.rb

```diff
@@ -86,7 +86,7 @@ def upload_file_in_chunks(file_path)
           filechunk.write(chunk)
           filechunk.flush()
           actual_chunk_size = File.size(filechunk)
-          response = @uploads_api.update(upload_href, content_range(offset, offset + actual_chunk_size -1, total_size), filechunk)
+          response = @uploads_api.update(content_range(offset, offset + actual_chunk_size -1, total_size), upload_href, filechunk)
```

Comment:
Should we use keyword arguments here?

Reply:
I tried, but as I'm don't know ruby, I think I did something wrong and it didn't work

Reply:
have you tried it like this?
```suggestion
          response = @uploads_api.update(content_range: content_range(offset, offset + actual_chunk_size -1, total_size), upload_href: upload_href, filechunk: filechunk)
```

Reply:
Nope! I tried:
```ruby
content_range_value = content_range(offset, offset + actual_chunk_size -1, total_size)
response = @uploads_api.update(content_range: content_range_value, upload_href: upload_href, filechunk: filechunk)
```

Reply:
I tried: c2f478a  got: https://travis-ci.com/github/pulp/pulpcore/jobs/362256353#L2545
I tried: b34e182 got: https://travis-ci.com/github/pulp/pulpcore/jobs/362258414#L2544

Reply:
Well, if no real rubyist wants to jump in, we should not be blocking on this.

Reply:
The method doesn't accept keyword args:

```ruby
def update(content_range, upload_href, file, opts = {})
  # ...
end
```

It would need to be `def update(content_range:, upload_href:, file:, opts = {})`.